host/tests/gatt.rs

```diff
@@ -41,20 +41,20 @@ async fn gatt_client_server() {
         // Random starting value to 'prove' the incremented value is correct
         let mut value: [u8; 1] = [rand::prelude::random(); 1];
         let mut expected = value[0].wrapping_add(1);
-        let mut svc = table.add_service(Service::new(0x1800));
-        let _ = svc.add_characteristic_ro(0x2a00, id);
-        let _ = svc.add_characteristic_ro(0x2a01, &appearance[..]);
+        let mut svc = table.add_service(Service::new(0x1800), None);
+        let _ = svc.add_characteristic_ro(0x2a00, id, None);
+        let _ = svc.add_characteristic_ro(0x2a01, &appearance, None);
```

Comment:
can this be written as
```
let mut svc = table.add_service(Service::new(0x1800), None);
svc.add_characteristic_ro(0x2a00, id, None);
svc.add_characteristic_ro(0x2a01, &appearance, None);
```
Similar to how these Builder patterns look?
https://www.shuttle.dev/blog/2022/06/09/the-builder-pattern